src/style.css

```
@@ -1 +1,80 @@
-/* add styles here */
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  width: 96px;
+  height: 116px;
+  align-items: flex-start;
+}
+
+.stars {
+  display: flex;
+  margin-right: 4px;
+}
+
+.stars__star {
+  display: flex;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 .stars__star:nth-child(n + 2) {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2 .stars__star:nth-child(n + 3) {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--3 .stars__star:nth-child(n + 4) {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(n + 5) {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url("images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

please change a selector not to duplicate style properties for each row 
example
```suggestion
.stars--2 .stars__star:nth-child(-n + 2) ,
.stars--3 .stars__star:nth-child(-n + 3) ,
.stars--4 .stars__star:nth-child(-n + 4) ,
.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url("images/star-active.svg");
}
```